lines/transformer/transform.py

```diff
@@ -0,0 +1,78 @@
+"""
+Rundown API response transformer mechanism for creating temporal updates.
+
+Generator of temporal records for `lines.producer.creator`
+
+(e.g.)
+    Temporal records represented by key: value pairs
+    NOTE: this must contain an temporal element
+    in this case the field: "event_date" from API response
+
+    "timestamp": "2020-02-02T23:30:00Z" is referenced as temporal element
+
+    {
+        "key": {
+            "eventId": "9a35b8986a76ea364be331cb453ec",  # string
+            "sportId": 2  # integer
+        },
+        "data": {
+            "timestamp": "2020-02-01T17:22:12Z",
+            "eventDate": "2020-02-02T23:30:00Z",
+            ...
+        }
+    }
+"""
```

Comment:
Should we use `jsonschema` to validate fields/payload? Link [here](https://python-jsonschema.readthedocs.io/en/stable/).